cmake/find/lz4.cmake

```diff
@@ -0,0 +1,27 @@
+option (USE_LZ4 "Set to FALSE to use system lz4 library instead of bundled" ${NOT_UNBUNDLED})
+
+if(NOT EXISTS "${ClickHouse_SOURCE_DIR}/contrib/lz4/lib/lz4.h")
+    if(USE_LZ4)
+        message(WARNING "submodule contrib/lz4 is missing. to fix try run: \n git submodule update --init --recursive")
+        message (${RECONFIGURE_MESSAGE_LEVEL} "Can't find internal lz4 library")
+        set(USE_LZ4 0)
+    endif()
+    set(MISSING_INTERNAL_LZ4_LIBRARY 1)
+endif()
+
+if (NOT USE_LZ4)
```

Comment:
I don't think clickhouse is ready to work / build w/o LZ4.

Reply:
Ok - In that case, I guess that I'll always build it with Lz4.